_posts/2017-02-18-digest-authentication-with-ruby-on-rails.md

```diff
@@ -0,0 +1,177 @@
+---
+layout: post
+title: "Digest: Authentication with Ruby on Rails"
+date: 2017-02-16 12:00
+author: Herbert Ilhan Tanujaya
+categories: [Digest]
+---
+
+## Introduction
+Nowadays, authentication systems are present in nearly all websites (with at
```

Comment:
Nowadays, nearly all websites have an authentication system in place, using usernames and passwords.